lib/omise/charge.ex

```diff
@@ -254,12 +257,21 @@ defmodule Omise.Charge do
   ## Examples
 
       Omise.Charge.capture("chrg_test_4xso2s8ivdej29pqnhz")
+      Omise.Charge.capture("chrg_test_4xso2s8ivdej29pqnhz",key: "skey_test_123")
+      Omise.Charge.capture("chrg_test_4xso2s8ivdej29pqnhz",[capture_amount: 3000])
+      Omise.Charge.capture("chrg_test_4xso2s8ivdej29pqnhz",[key: "skey_test_123"],[capture_amount: 3000])
 
   """
-  @spec capture(String.t(), Keyword.t()) :: {:ok, t} | {:error, Omise.Error.t()}
-  def capture(id, opts \\ []) do
-    opts = Keyword.merge(opts, as: %__MODULE__{})
-    post("#{@endpoint}/#{id}/capture", [], opts)
+  @spec capture(String.t(), Keyword.t(), Keyword.t()) :: {:ok, t} | {:error, Omise.Error.t()}
+  def capture(id, opts \\ [], params \\ []) do
+    contains_secret_key= Enum.any?(opts, fn {key, _} -> key == :key end)
+    {newParams, newOpts} = if opts != [] and !contains_secret_key do
```

Comment:
Note that in Elixir, we generally use snake_case for variable names, not camelCase.